homework-05/script.js

```diff
@@ -0,0 +1,14 @@
+let hours = + prompt('Enter the number of hours:');
+let secondsTotal;
+const minuts = 60;
+const seconds = 60;
```

Comment:
```suggestion
const MINUTES_IN_HOUR = 60;
const SECONDS_IN_MINUTE = 60;
```

назви констант пишуть великими літерами
назви треба робити якомога зрозумілішими